### ISSUE TITLE
Quand on clique sur le pin d'un lien, le lien vers le lieu ne fonctionne pas

### ISSUE THREAD

**Issue by nfavrod** [Skrattoune/OpenRepairPlatform#12]
<img width="731" alt="Capture d’écran 2022-03-11 à 00 52 52" src="https://user-images.githubusercontent.com/1118004/157775068-32a70b15-5bda-4a4f-a9b6-a41bdf75764c.png">

**Comment by Skrattoune** [comment]
sur : https://dev-openrepairplateform.repaircafes.ch/event/

**Comment by nfavrod** [comment]
Aussi en lien avec les issues #25 et #7

**Comment by Skrattoune** [comment]
non, pas de liens à priori, mais pas le temps maintenant de bosser là-dessus
ce n'est pas une issue qui nous empêche de rentrer en prod et à utiliser l'app avec le grand public
